### DISCUSSION TITLE
sync with gpodder client ?

### DISCUSSION THREAD

**Discussion by ghost** [thrillfall/nextcloud-gpodder#80] [General]
I'm looking for an alternative to gpodder.net which is often overloaded and unreachable.
Is this app doing the same ?
I've gpodder as a client on several PCs and it's the podcast client I prefer. Could they be synchronised via this nextcloud-gpodder app ?
Thanks

**Comment by JonOfUs** [comment]
Hi,
currently gpodder is not supported, so this will not work due to small api url differences. The supported clients are listed in the README of this project (Antennapod and KDE Kasts).
Someone has to add support for the Nextcloud app to the gpodder client, we cannot do anything in this project to support it.

**Comment by lixuuu** [comment]
Quite surprising that the gpodder client does not use the same api format as Antennapod. Too bad 2 great free software don't speak the same language. I am really surprised. 

I didn't know the DKE Kasts podcast client. Being a KDE desktop user, I will try KDE Kasts in this case. 

We have 2 programs Antennapod and KDE Kasts which speak the same language. So it'd be logical that gppoder adapts to the language of Antennapod and KDE Kasts
